src/features/todos.ts

```diff
@@ -1,8 +1,12 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { createSlice, PayloadAction } from '@reduxjs/toolkit';
 import { Todo } from '../types/Todo';
 
 export const todosSlice = createSlice({
   name: 'todos',
   initialState: [] as Todo[],
-  reducers: {},
+  reducers: {
+    set(todos: Todo[], { payload }: PayloadAction<Todo[]>): Todo[] {
```

Comment:
The reducer function `set` should have the state as the first parameter and the action as the second parameter. Correct the function signature to: `set(state: Todo[], action: PayloadAction<Todo[]>): Todo[] { return [...action.payload]; }`.